README.md

```diff
@@ -1,37 +1,71 @@
 # Discourse Operator
 
-A juju charm deploying and managing Discourse on Kubernetes, configurable to
-use a PostgreSQL backend.
+A juju charm deploying and managing Discourse on Kubernetes. Discourse is the
+100% open source discussion platform. Use it as a mailing list, discussion
+forum or long-form chat room.
 
-## Overview
+This charm simplifies initial deployment and "day N" operations of Discourse
+on Kubernetes, such as scaling the number of instances, integration with SSO,
+access to S3 for redundant file storage and more. It allows for deployment on
+many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) to
+[Charmed Kubernetes](https://ubuntu.com/kubernetes) to public cloud Kubernetes
```

Comment:
A very small remark:
I'd rather write: "from MicroK8S or Charmed Kubernetes to public cloud..."
Any way to avoid repeating "to"